From da48e745aab98f0081bcc1ca7792b011cecfb237 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Sun, 28 Aug 2016 15:04:36 -0700
Subject: [PATCH 19/19] no wayland

Remove commented out entry in config file about enabling/disabling
Wayland to avoid confusion, since we don't compile Wayland support
into the binaries at all.
---
 data/gdm.conf-custom.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/gdm.conf-custom.in b/data/gdm.conf-custom.in
index c9e2864..438d518 100644
--- a/data/gdm.conf-custom.in
+++ b/data/gdm.conf-custom.in
@@ -1,8 +1,6 @@
 # GDM configuration storage
 
 [daemon]
-# Uncoment the line below to force the login screen to use Xorg
-#WaylandEnable=false
 
 [security]
 
-- 
2.7.4

